src/plugins/video/video.yml

```diff
@@ -37,3 +37,6 @@ width:
 copyrightInfos:
   en: Copyright infos
   de: Copyright Infos
+posterImageUrl:
+  en: Preview image URL
+  de: Vorschau Bild URL
```

Comment:
We need hyphens between the words: "Vorschau-Bild-URL"

Reply:
Ge-hyphenated, thank you